example/App.js

```diff
@@ -6,15 +6,48 @@
 
 import { createAppContainer } from 'react-navigation';
 import { createBottomTabNavigator } from 'react-navigation-tabs';
+import { createStackNavigator } from 'react-navigation-stack';
 
 import HomeScreen from "./screens/HomeScreen";
 import SettingsScreen from "./screens/SettingsScreen";
+import MessageListScreen from "./screens/MessageCenterScreen";
+import MessageDetailsScreen from "./screens/MessageScreen";
```

Comment:
I suggest making these more consistent, e.g.:
```suggestion
import MessageCenterScreen from "./screens/MessageCenterScreen";
import MessageScreen from "./screens/MessageScreen";
```